app/src/main/java/com/eventyay/organizer/core/auth/login/LoginViewModel.java

```diff
@@ -76,41 +77,51 @@ private void encryptUserCredentials() {
         sharedPreferenceModel.saveString(PREF_USER_PASSWORD, encryptedPassword);
     }
 
-    //for logging into the app
+    // for logging into the app
     public void login() {
-        compositeDisposable.add(loginModel.login(login)
-            .doOnSubscribe(disposable -> progress.setValue(true))
-            .doFinally(() -> progress.setValue(false))
-            .subscribe(() -> {
-                    encryptUserCredentials();
-                    actionLogin.call();
-                },
-                throwable -> {
-                    if (throwable instanceof HttpException) {
-                        int errorCode = ((HttpException) throwable).code();
-
-                        if (errorCode == 401)
-                            error.setValue("Please check the credentials you have entered");
-                        else
-                            error.setValue(ErrorUtils.getMessage(throwable).toString());
-                    } else {
-                        error.setValue(ErrorUtils.getMessage(throwable).toString());
-                    }
-                }));
+        compositeDisposable.add(
+                loginModel
+                        .login(login)
+                        .doOnSubscribe(disposable -> progress.setValue(true))
+                        .doFinally(() -> progress.setValue(false))
+                        .subscribe(
+                                () -> {
+                                    encryptUserCredentials();
+                                    actionLogin.call();
+                                },
+                                throwable -> {
+                                    if (throwable instanceof HttpException) {
+                                        int errorCode = ((HttpException) throwable).code();
+
+                                        if (errorCode == 401)
+                                            error.setValue(
+                                                    "Please check the credentials you have entered");
+                                        else
+                                            error.setValue(
+                                                    ErrorUtils.getMessage(throwable).toString());
+                                    } else {
+                                        error.setValue(ErrorUtils.getMessage(throwable).toString());
+                                    }
+                                }));
     }
 
     public void setBaseUrl() {
-        String baseUrl = sharedPreferenceModel.getString(Constants.SHARED_PREFS_BASE_URL,
-            BuildConfig.DEFAULT_BASE_URL);
+        String baseUrl =
+                sharedPreferenceModel.getString(
+                        Constants.SHARED_PREFS_BASE_URL, BuildConfig.DEFAULT_BASE_URL);
```

Comment:
This formatting looks weird. Doesn't it? @liveHarshit

`sharedPreferenceModel.getString(` could have been in the same line as `String baseUrl =`.

Reply:
Yes

Reply:
I used the Android Open Source Project filter on googleJavaFormat() in the plugin which is causing such kind of formatting. Should I remove the AOSP filter?